.github/workflows/e2ePerformanceTests.yml

```
@@ -94,7 +109,7 @@ jobs:
           ARTIFACT_NAME: delta-apk-${{ needs.prepare.outputs.DELTA_REF }}
 
   runTestsInAWS:
-    runs-on: ubuntu-20.04-64core
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-20.04-64core
```
Modified for testing purposes, revert change before merging to `main`.